src/server/models/listItem/providers/TranslatorsInterpreters.ts

```diff
@@ -0,0 +1,92 @@
+import { TranslatorInterpreterListItemGetObject, ServiceType } from "server/models/types";
+import { getPlaceGeoPoint } from "./../geoHelpers";
+import { startCase, toLower } from "lodash";
+import { logger } from "server/services/logger";
+import { prisma } from "server/models/db/prisma-client";
+import { fetchPublishedListItemQuery } from "server/models/listItem/providers/helpers";
+
+export async function findPublishedTranslatorsInterpretersPerCountry(props: {
+  countryName?: string;
+  region?: string | "";
```

Comment:
I don't know if a union type is necessary here if the right part of the union is an empty string. This should be enough
```suggestion
  region?: string;
```

Reply:
this could potentially be undefined, hence empty string will come into use then.

Reply:
empty string is not the same as undefined (javascript funkiness), beware :)

Reply:
Wait, is an empty sting the same as undefined in other languages 😱 

@alisalaman-cyb an, for undefined in ts the type is `undefined` but using the question mark to signify an optional parameter should make it `string | undefined` by default 🤔

Reply:
nop, more to do with falsey checks. if anything, javascript is the only language that will evaluate empty string and undefined the same way (false). 

`""` will evaluate to false, the same way that `undefined` does. 

however, const { page = 1 } = params will *only* work in undefined cases. not empty string cases. 

>this could potentially be undefined, hence empty string will come into use then.
`undefined`, `""` mean different things and are different types